streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StatefulProcessorNode.java

```diff
@@ -69,55 +80,4 @@ public void writeToTopology(final InternalTopologyBuilder topologyBuilder) {
         }
     }
 
-    public static <K, V> StatefulProcessorNodeBuilder<K, V> statefulProcessorNodeBuilder() {
-        return new StatefulProcessorNodeBuilder<>();
-    }
-
-    public static final class StatefulProcessorNodeBuilder<K, V> {
-
-        private ProcessorParameters processorSupplier;
-        private String nodeName;
-        private boolean repartitionRequired;
-        private String[] storeNames;
-        private StoreBuilder<? extends StateStore> storeBuilder;
-
-        private StatefulProcessorNodeBuilder() {
-        }
-
-        public StatefulProcessorNodeBuilder<K, V> withProcessorParameters(final ProcessorParameters processorParameters) {
-            this.processorSupplier = processorParameters;
-            return this;
-        }
-
-        public StatefulProcessorNodeBuilder<K, V> withNodeName(final String nodeName) {
-            this.nodeName = nodeName;
-            return this;
-        }
-
-        public StatefulProcessorNodeBuilder<K, V> withStoreNames(final String[] storeNames) {
-            this.storeNames = storeNames;
-            return this;
-        }
-
-        public StatefulProcessorNodeBuilder<K, V> withRepartitionRequired(final boolean repartitionRequired) {
-            this.repartitionRequired = repartitionRequired;
-            return this;
-        }
-
-        public StatefulProcessorNodeBuilder<K, V> withStoreBuilder(final StoreBuilder<? extends StateStore> storeBuilder) {
-            this.storeBuilder = storeBuilder;
-            return this;
-        }
-
-        public StatefulProcessorNode<K, V> build() {
-            return new StatefulProcessorNode<>(
-                nodeName,
-                processorSupplier,
-                storeNames,
-                storeBuilder,
-                repartitionRequired
-            );
-
-        }
-    }
```

Comment:
This is an internal class, and the constructor is quite simple. Having a builder for it seems like extra code for no real gain. Plus, it makes it more difficult for callers to tell what arguments are expected.